provider/testing/resource.go

```diff
@@ -53,28 +53,37 @@ func TestResource(t *testing.T, resource ResourceTestCase) {
 	}
 
 	l := testlog.New(t)
-	l.SetLevel(hclog.Debug)
+	l.SetLevel(hclog.Info)
 	resource.Provider.Logger = l
 	tableCreator := migration.NewTableCreator(l, schema.PostgresDialect{})
-	if err := tableCreator.CreateTable(context.Background(), conn, resource.Table, nil); err != nil {
-		assert.FailNow(t, fmt.Sprintf("failed to create tables %s", resource.Table.Name), err)
-	}
 
-	if err := deleteTables(conn, resource.Table); err != nil {
-		t.Fatal(err)
+	for _, table := range resource.Provider.ResourceMap {
+		if err := tableCreator.CreateTable(context.Background(), conn, table, nil); err != nil {
+			assert.FailNow(t, fmt.Sprintf("failed to create tables %s", table.Name), err)
+		}
+		if err := deleteTables(conn, table); err != nil {
+			t.Fatal(err)
+		}
 	}
```

Comment:
Does this delete the data or the entire table? If it is deleting the data can we call it `truncateTables`?